designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Developer
+version: 4.5
+---
+# Add page- Updates
+
+Updates to the Add page include switching the order of the resources and pipelines section so resources come before pipelines.
+One more subtle update to the Add page is that the ‘Show pipeline visualization’ will stay open even if the user selects a new builder image or resource during their session.
+
+## Upload from Git- Updates
+
+![image](https://github.com/lwrigh/openshift-origin-design/blob/master/designs/developer/add-45/Pipelines-update/img/import-from-git-1.png)
+
+A builder image is selected. When a builder image has not been selected yet, the ‘Pipelines’ section will display an info alert that says “Select a builder image and resource to see if there is a pipeline template available for this runtime.”
+
+**Note:** ‘Resources’ has been moved to the section before ‘Pipelines’.
+
+![image](https://github.com/lwrigh/openshift-origin-design/blob/master/designs/developer/add-45/Pipelines-update/img/import-from-git-2.png)
+
+A builder image has been selected and the pipelines section of the Add page reacts and displays a checkbox to add a pipeline. The checkbox can be selected if the user wants to add a pipeline to their project.
+
+![image](https://github.com/lwrigh/openshift-origin-design/blob/master/designs/developer/add-45/Pipelines-update/img/import-from-git-3.png)
+
+To show the pipeline visualization, the caret or label can be clicked to expand it. 
+
+![image](https://github.com/lwrigh/openshift-origin-design/blob/master/designs/developer/add-45/Pipelines-update/img/import-from-git-4.png)
+
+To view more information about each of the individual pipeline sections, the user can hover over each section to get more details about it. The pipeline visualization label changes to ‘Hide pipeline visualization’ if the pipeline visualization is expanded.
+
+**Note:** This is exactly like what is currently implemented.
```

Comment:
I don't think we need this Note in the docs...

Reply:
I can get rid of it.